src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -183,12 +194,13 @@ inline void FUNC(fc_bf_tiled_kernel_default)(
     , FUSED_OPS_DECLS
 #endif
 ) {
-#if USE_SLM
-    uint gid = (uint)get_group_id(0);
-    uint local_id = (uint)get_local_id(2);
-#else
-    uint gid = (uint)get_group_id(0);
-#endif
+    #if USE_SLM
+        uint gid = (uint)get_group_id(0);
+        uint local_id = (uint)get_local_id(2);
+    #else
+        uint gid = (uint)get_group_id(0);
+    #endif
```

Comment:
nit: the style of indent is quite mixed in cl files. But let's not make unnecessary changes..

Reply:
Modified.